packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx

```diff
@@ -150,15 +181,37 @@ export default function ApiKeyDialogForm({
               </div>
               <Controller
                 name="expiresAt"
-                render={({ field: { onChange, value } }) => (
-                  <DatePicker
-                    disabled={watchNeverExpires || !!defaultValues}
-                    minDate={new Date()}
-                    date={value}
-                    onDatesChange={onChange}
+                render={({ field: { onChange, value } }) => {
+                  const defaultValue = expiresAtOptions[1]
+                  console.log(defaultValue,value,expiresAtOptions);
+                  
+                  return(
+                  <SelectField
+                    styles={{
+                      singleValue: (baseStyles) => ({
+                        ...baseStyles,
+                        fontSize: "14px",
+                      }),
+                      option: (baseStyles) => ({
+                        ...baseStyles,
+                        fontSize: "14px",
+                      }),
+                    }}
+                    isDisabled={watchNeverExpires || !!defaultValues}
+                    containerClassName="data-testid-field-type"
+                    options={expiresAtOptions}
+                    onChange={(option) => {
+                      if (!option) {
+                        return;
+                      }
+                      onChange(option.value);
+                      setExpiryDate(option.value)
+                    }}
+                    defaultValue={defaultValue}
                   />
-                )}
+                )}}
               />
+              <span className="text-subtle text-xs mt-2">The API key will expire on <span className="font-bold"> {dayjs(expiryDate).format('DD-MM-YYYY')}</span></span>
```

Comment:
I guess this text should be translated, right?

Reply:
@devj3ns I fixed it can you review it!! ASAP